src/utils/Enums/FilterBy.ts

```diff
@@ -0,0 +1,5 @@
+export enum FilterBy {
+  ALL = 'all',
+  ACTIVE = 'active',
+  COMPLETED = 'completed',
```

Comment:
```suggestion
export enum FilterBy {
  All = 'all',
  Active = 'active',
  Completed = 'completed',
```
Enum properties should be in PascalCase, not upper case